draft-ietf-quic-multipath.md

```diff
@@ -1405,11 +1410,10 @@ such as e.g. for packet number handling, only after path validation has successf
 
 ## Request Forgery with Spoofed Address
 
-The path validation mechanism as specified in {{Section 8.2. of QUIC-TRANSPORT}} for migration is used
-unchanged for initiation of new paths in this extension. Respectively the security considerations
-on source address spoofing as outlined in {{Section 21.5.4 of QUIC-TRANSPORT}} equally apply.
-Similarly, the anti-amplification limits as specified in {{Section 8 of QUIC-TRANSPORT}} need to be
-followed to limit the amplification risk.
+The path validation mechanisms and anti-amplification limits specified in
+{{Section 8 of QUIC-TRANSPORT}} for migration also apply to initiation of new
+paths in this extension. Therefore, the security considerations on source
+address spoofing in {{Section 21.5.4 of QUIC-TRANSPORT}} equally apply.
 
```

Comment:
If we have a reference there, maybe we do not need to also have it in the path initiation section.

Reply:
Perhaps, though I suspect putting it there is more likely to catch the attention of someone implementing that flow.